src/OpenTelemetry.Instrumentation.Runtime/MeterProviderBuilderExtensions.cs

```diff
@@ -11,15 +13,24 @@ namespace OpenTelemetry.Metrics;
 /// </summary>
 public static class MeterProviderBuilderExtensions
 {
+#if NET9_0_OR_GREATER
+    private const string DotNetRuntimeMeterName = "System.Runtime";
+#endif
+
     /// <summary>
     /// Enables runtime instrumentation.
     /// </summary>
     /// <param name="builder"><see cref="MeterProviderBuilder"/> being configured.</param>
     /// <returns>The instance of <see cref="MeterProviderBuilder"/> to chain the calls.</returns>
     public static MeterProviderBuilder AddRuntimeInstrumentation(
         this MeterProviderBuilder builder) =>
+#if NET9_0_OR_GREATER
+        builder.AddMeter(DotNetRuntimeMeterName);
+#else
```

Comment:
Please make this a runtime check (as ASP.NET Core). In OTel .NET Auto we are shipping binary in the lowest supported version (in this case it will be .NET8).

Reply:
Ah, gotcha. Updated, which simplifies the rest of the changes also.